src/mh-core.jl

```diff
@@ -157,6 +173,7 @@ function AbstractMCMC.step(
         transition = AdvancedMH.transition(spl, model, init_params)
     end
 
+    trackstep!(spl.proposal, transition.params)
```

Comment:
The `step` implementations are written for generic `AbstractTransition`s and `MHSampler`, so neither `spl.proposal` nor `transition.params` might exist.